File: CPPSolutions/LeetCode/String/M2864_最大二进制奇数.cpp
#include <bits\stdc++.h>

using namespace std;

class Solution {
public:
    string maximumOddBinaryNumber(string s) {
        int n = s.length(), cnt = count(s.begin(), s.end(), '1');
        return string(cnt - 1, '1') + string(n - cnt, '0') + '1';
    }
};

File: GolangSolutions/String/M2864_最大二进制奇数.go
/*
	@author Hyper
	@date 2024/03/13
	@file M2864_最大二进制奇数.go
	Golang可以直接调用strings.Count()和strings.Repeat()完成
*/

package String

import "strings"

func maximumOddBinaryNumber(s string) string {
	n, cnt := len(s), strings.Count(s, "1")
	return strings.Repeat("1", cnt-1) + strings.Repeat("0", n-cnt) + "1"
}
